tests/data/issues_array_v1.1.2.json

```
@@ -21,10 +21,8 @@
       "disease__treated: 'T' is not of type 'boolean'": [
         "BM01_16dpp_CTAAGCAGTGGT"
       ],
-      "disease__time_since_onset: '36|three|1' is not of type 'array'": [
-        "BM01_16dpp_CGGTAAACCATT"
-      ],
-      "disease__time_since_onset: 'zero' is not of type 'array'": [
+      "'disease__time_since_onset' is a dependency of 'disease__time_since_onset__unit'": [
```

"depends on" seems like a simpler equivalent to "is a dependency of":

```suggestion
      "'disease__time_since_onset' depends on 'disease__time_since_onset__unit'": [
```

If you have qualms or think it'd be more than a 30-second change across files, please ignore this suggestion!

---

I agree the message is hard to parse. Unfortunately this one is generated by the JSON schema validator so it isn't something we can easily modify :(